lib/winston-syslog.js

```diff
@@ -146,7 +145,7 @@ Syslog.prototype.log = function (level, msg, meta, callback) {
   syslogMsg = this.producer.produce({
     severity: level,
     host:     this.localhost,
-    appName:   this.appId || process.title,
+    appName:   this.appName || process.title,
```

Comment:
I wonder if removing this line would also work, as the `appName` in the `producer` has already been set in the constructor.